app/models/work.rb

```diff
@@ -0,0 +1,37 @@
+class Work < ApplicationRecord
+  has_many :votes
+  has_many :users, through: :votes
+
+  validates :category, presence: { message: "You must choose a category" }
+
+  validates :title, presence: { message: "Title can't be blank" }
+
+  def self.all_movies
+    where(category: "movie")
+  end
+
+  def self.all_books
+    where(category: "book")
+  end
+
+  def self.all_albums
+    where(category: "album")
+  end
+
+  def self.media_spotlight
+    all.max_by{ |w| w.votes.count }
+  end
+
+  def self.top_movies
+    where(category: "movie").sort_by{ |m| m.votes.count }.reverse.first(10)
```

Comment:
These methods could be DRYed up slightly by creating a generic method with a parameter.